docs/lib/datastore/fragments/android/getting-started.md

```diff
@@ -25,10 +33,16 @@ Next, add the following dependencies to your app `build.gradle`:
 
 ```groovy
 implementation 'com.amplifyframework:core:0.10.0'
+implementation 'com.amplifyframework:aws-api:0.10.0' // If using cloud sync
```

Comment:
Another note for you, @drochetti, and later -- I'm of the opinion that the getting started guide should be as prescriptive and direct as possible, leading a reader from knowing nothing to doing something useful in the category within minutes. Integrating the API category seems like it introduces more conceptual overhead than is worthwhile. I liked @swaminator's view on this that a getting started guide that builds the skeleton of a locally stored TODO list is useful and can connect the concepts well. Anyway, not for this pull, but wanted to raise as we're polishing.